creator/lib/widgets/carousel_with_dot.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:flutter/material.dart';
+
+class CarouselWithDotsPage extends StatefulWidget {
+  List<String> imgList;
+  CarouselWithDotsPage({this.imgList, Key key}) : super(key: key);
+
+  @override
+  _CarouselWithDotsPageState createState() => _CarouselWithDotsPageState();
+}
+
+class _CarouselWithDotsPageState extends State<CarouselWithDotsPage> {
+  int _current = 0;
+
+  Widget _bottomTextOfCarousel(String item) {
+    return Positioned(
+      bottom: 0.0,
+      left: 0.0,
+      right: 0.0,
+      child: Container(
+        decoration: decoration_of_carousel_text(),
+        padding: EdgeInsets.symmetric(
+          horizontal: 20,
+          vertical: 10,
+        ),
+        child: Text(
+          'No. ${widget.imgList.indexOf(item)} image',
+          style: TextStyle(
+            color: Colors.white,
+            fontSize: 12.0,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+      ),
+    );
+  }
+
+  BoxDecoration decoration_of_carousel_text() {
```

Comment:
please use camelCasing for method names and use the verbNoun format
```suggestion
  BoxDecoration _buildCarouselTextDecoration() {
```